src/frontend/citizen-portal/src/app/services/notice-of-dispute.service.ts

```diff
@@ -144,12 +149,17 @@ export class NoticeOfDisputeService {
   }
 
   createNoticeOfDispute(input: NoticeOfDispute): void {
-    input.disputant_birthdate = this.datePipe.transform(input.disputant_birthdate, "yyyy-MM-dd");
     input.issued_date = this.datePipe.transform(input.issued_date, "yyyy-MM-ddTHH:mm:ss");
-    input = this.splitGivenNames(input);  // break disputant names into first, second, third
+    input = this.splitDisputantGivenNames(input);  // break disputant names into first, second, third
+    input = this.splitContactGivenNames(input);  // break disputant names into first, second, third
     input = this.splitLawyerNames(input); // break lawyer names into first, second, surname
     input = this.splitAddressLines(input); // break address into line 1,2,3 by comma
 
+    // input.disputant_birthdate = "2001-01-01" // TODO: remove this once disputant birthdate removed from schema, API
+    input.dispute_counts.forEach(count => { // TODO: remove this once request_court_appearance removed from dispute count schema, API
+        count.request_court_appearance = input.request_court_appearance;
+    });
```

Comment:
can be removed, dispute stepper is updated

Reply:
done